spec/test_collector/minitest_plugin/trace_spec.rb

```diff
@@ -38,6 +45,12 @@
       expect(history_json).to include('347611.734956')
     end
 
+    it 'sets the failure_expanded' do
+      failure_expanded = trace.as_hash[:failure_expanded]
+      expect(failure_expanded).not_to be_empty
+      expect(failure_expanded[0][:expanded]).not_to include('test for invalid character')
+    end
```

Comment:
Do we also want to add tests to verify:
* `failure_reason` only contains the first line?
* `failure_expanded` contains the remaining lines?

Reply:
Should we add some tests for the case where the `failures` array has more than one failure in it? Is it possible to find/generate some legit data for this?